_staging/src/k8s.io/client-go/copy.sh

```diff
@@ -29,6 +29,7 @@ CLIENT_REPO_TEMP="${CLIENT_REPO}"/_tmp
 # working in the ${CLIENT_REPO_TEMP} so 'godep save' won't complain about dirty working tree.
 echo "creating the _tmp directory"
 mkdir -p "${CLIENT_REPO_TEMP}"
+cd ${CLIENT_REPO}
```

Comment:
All other changes are still LGTM (beyond bash which I refuse to be
responsible for understanding)

On Thu, Aug 4, 2016 at 1:32 PM, Chao Xu <notifications@github.com> wrote:

> In _staging/src/k8s.io/client-go/copy.sh
> <https://github.com/kubernetes/kubernetes/pull/29147#discussion_r73566895>
> :
>
> > @@ -29,6 +29,7 @@ CLIENT_REPO_TEMP="${CLIENT_REPO}"/_tmp
> >  # working in the ${CLIENT_REPO_TEMP} so 'godep save' won't complain about dirty working tree.
> >  echo "creating the _tmp directory"
> >  mkdir -p "${CLIENT_REPO_TEMP}"
> > +cd ${CLIENT_REPO}
>
> Done. Thanks.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29147/files/e7fccc2d9d0bd06a6e5ff96893221053b3457866..a58fce35f3ba60d9a24668b6e5f05422aa27b4d5#r73566895>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p3JtRFkGhiRTKNZWofI44y2T_ni2ks5qciI_gaJpZM4JPNRl>
> .
>